Astral Pathfinder/Models/Planet.hpp

```diff
@@ -15,28 +15,27 @@
 #include "parameters.h"
 
 class Planet {
+
 public:
-    Planet();
-    ~Planet();
+  void initHomeworld();
+  void initPlanet();
   
-    void initHomeworld();
-    void initPlanet();
-    int getPositionX();
-    int getPositionY();
-    int getDeposits();
-    int getFertility();
+  int getPositionX() { return position.y; };
```

Comment:
Let's pretend you didn't mix up which function returns what ;)

Reply:
Also, why not this.position.x? I'm sure it's fine either way but wouldn't 'this' be safer?